airbyte-webapp/src/hooks/services/ConnectionForm/ConnectionFormService.tsx

```diff
@@ -33,10 +33,14 @@ export const tidyConnectionFormValues = (
   values: FormikConnectionFormValues,
   workspaceId: string,
   mode: ConnectionFormMode,
+  allowSubOneHourCronExpressions: boolean,
```

Comment:
I tried to avoid extending this function with yet another argument, but it would have required refactoring quite a lot of the connection form schema.